components/form-builder/preview/Form.tsx

```diff
@@ -170,39 +149,19 @@ const InnerForm: React.FC<InnerFormProps> = (props) => {
           {errorList}
         </Alert>
       )}
-      {formStatusSubmitted && props.isPreview && (
-        <>
-          <Link
-            onClick={(e) => {
-              e.preventDefault();
-              props.setStatus("None");
-            }}
-            href="#"
-          >
-            <BackArrowIcon />
-            Back to form preview
-          </Link>
-          <Markdown options={{ forceBlock: true }}>
-            {form.endPage
-              ? form.endPage[localizeField(LocalizedElementProperties.DESCRIPTION)]
-              : ""}
-          </Markdown>
-        </>
-      )}
       {/**
        * method attribute needs to stay here in case javascript does not load
        * otherwise GET request will be sent which will result in leaking all the user data
        * to the URL
        */}
-      {!formStatusSubmitted && (
+      {
         <>
-          <div className="gc-richText">
-            <Markdown options={{ forceBlock: true }}>
-              {form.introduction
-                ? form.introduction[localizeField(LocalizedElementProperties.DESCRIPTION)]
-                : ""}
-            </Markdown>
-          </div>
+          <RichText>
```

Comment:
Should be able to use && here

form.introduction && <RichText>form.introduction[localizeField(LocalizedElementProperties.DESCRIPTION)</RichText>